### PR TITLE
docs(how-to-guides): add how to fix dependent package versions

### PR DESCRIPTION
**Pull request by IshitaTakeshi**
## Description

Add how to fix the dependent package versions and the package version currently developing

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by IshitaTakeshi** [comment]
I'm going to fix spelling

File: docs/how-to-guides/fixing-dependent-package-versions.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Fixing dependent package versions
+
+Autoware manages dependent package versions in autoware.repos.
+For example, let's say you make a branch in autowre.universe and add new features.
```
Comment (kenji-miyake) [inline docs/how-to-guides/fixing-dependent-package-versions.md]:
```suggestion
For example, let's say you make a branch in autoware.universe and add new features.
```

File: docs/how-to-guides/fixing-dependent-package-versions.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Fixing dependent package versions
```
Comment (kenji-miyake) [inline docs/how-to-guides/fixing-dependent-package-versions.md:1]: Please also update `.pages` or the index page.

Comment (IshitaTakeshi) [reply docs/how-to-guides/fixing-dependent-package-versions.md:1]: Resolved in [7d27537](https://github.com/autowarefoundation/autoware-documentation/pull/295/commits/7d27537fa40a1f52d53fa99c3bb101b57a214b2b)

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**